### ISSUE TITLE
Glitch, animals not reaching butcher's shop.

### ISSUE THREAD

**Issue by Nightshae** [TangoTek/TekTopia-Community#403]
**Describe the bug**
Tango, you mentioned that you had an issue with the butcher while filming.  I watched and observed the following happening twice: The animals took enough damage going around buildings that they died before reaching the butcher's shop.  Essentially, they kept glitching into walls while being led.  I observed this twice, once with a cow, and once with a sheep.  In both cases the butcher stood still while I picked up lead ropes, meat and leather/wool, then simply walked away off to the library to read.

If you want to go to the village and observe this behavior yourself, feel free to do so.  The butchery is under the kitchen part of my tavern.  To find it, go out the sheep pen, past the cows, then the smithy, down the hill, then around the corner to the left.  The entrance to the butcher's shop is just up the next hill, under the kitchen.  They are not connected.

Unfortunately, the action happened too fast and I was unable to get screen shots.

** Server/Location **
What server were you playing on? Main(Survival) or Dev (Creative)? What If Server #1
What are the world coordinates of your town hall marker?  1556 70 -3281

**Comment by JohnyDL** [comment]
Yeah I'm seeing this happen even without them being in walls only passing through fences. Maybe only TP the animals to the pathing grid is a good plan?
